openapiart/openapiartprotobuf.py

```diff
@@ -167,11 +223,15 @@ def _get_field_type(self, property_name, openapi_object):
                         return "float"
                 return "float"
             if type == "array":
-                item_type = self._get_field_type(property_name, openapi_object["items"])
+                item_type = self._get_field_type(
+                    property_name, openapi_object["items"]
+                )
                 format = openapi_object.get("format")
                 min = openapi_object.get("minimum")
                 max = openapi_object.get("maximum")
-                if (min is not None and min > 2147483647) or (max is not None and max > 2147483647):
+                if (min is not None and min > 2147483647) or (
```

Comment:
Could you please let me know linter will automatically modify and update in github repository (files within openapiart)/ Need to change locally and push those change.

Reply:
black will auto format, but errors from cornflakes we have to change locally and push. If any linting error with cornflakes, CI/CD would fail